cmd/eksctl-anywhere/cmd/get.go

```diff
@@ -11,3 +25,49 @@ var getCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(getCmd)
 }
+
+func preRunGetPackages(cmd *cobra.Command, args []string) error {
+	if !features.IsActive(features.CuratedPackagesSupport()) {
+		return fmt.Errorf("this command is currently not supported")
+	}
+	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+		if err := viper.BindPFlag(flag.Name, flag); err != nil {
+			log.Fatalf("Error initializing flags: %v", err)
+		}
+	})
+	return nil
+}
+
+func getResources(ctx context.Context, resourceType string, output string, args []string) error {
+	kubeConfig := os.Getenv(kubeconfigEnvVariable)
+
+	deps, err := createKubectl(ctx)
+	if err != nil {
+		return fmt.Errorf("unable to initialize executables: %v", err)
+	}
+	kubectl := deps.Kubectl
+
+	params := []executables.KubectlOpt{executables.WithKubeconfig(kubeConfig), executables.WithArgs(args), executables.WithNamespace(constants.EksaPackagesName)}
+	if output != "" {
+		params = append(params, executables.WithOutput(output))
+	}
+	packages, err := kubectl.GetResources(ctx, resourceType, params...)
+	if err != nil {
+		fmt.Print(packages)
```

Comment:
I guess packages will never be nil?

Reply:
Yes that's correct. It might be nil only if err is not nil in which case, we are already returning the err.